frontend/sozisel-app/src/components/ParticipantActiveSession/Modules/QuizEvent/ParticipantQuizEvent.tsx

```diff
@@ -78,23 +79,25 @@ export default function ParticipantQuizEvent({
   }, [dispatch, quiz]);
 
   return (
-    <Paper className="ParticipantQuizEvent" elevation={2}>
-      <div className="header">
-        <Typography variant="h5" className="headerText">
-          <EventOutlinedIcon className="eventIcon" />
-          {event.name}
-        </Typography>
-        <Typography variant="h5">
-          {questionNumber + 1}/{quiz.quizQuestions.length}
-        </Typography>
+    <SoziselCard>
```

Comment:
Can you rename to something not containing `Sozisel` in component name? `ShadowBox` seems cool.